components/form.tsx

```diff
@@ -19,7 +19,7 @@ const Form = ({ formData, onCancel, onSubmit }: FormProps) => {
     const [errors, setErrors] = useState<StringKeyValue>({});
 
     const handleChange = (event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
-        const { name: formName, value } = event?.target;
+        const { name: formName, value } = event?.target || {};
```

Comment:
Is this to prevent `formName` and `value` trying to destructure from a possible `undefined` value?